src/main/java/org/apache/accumulo/testing/randomwalk/shard/Grep.java

```diff
@@ -88,13 +88,11 @@ public void visit(State state, RandWalkEnv env, Properties props) throws Excepti
     bs.close();
 
     if (!documentsFoundInIndex.equals(documentsFoundByGrep)) {
-      throw new Exception(
-          "Set of documents found not equal for words " + Arrays.asList(words).toString() + " "
-              + documentsFoundInIndex + " " + documentsFoundByGrep);
+      throw new Exception("Set of documents found not equal for words " + Arrays.asList(words) + " "
```

Comment:
```suggestion
      throw new Exception("Set of documents found not equal for words " + Arrays.toString(words) + " "
```
Does the same thing without having to first convert it to List.